src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -2031,6 +2031,15 @@ void Eltwise::prepareParams() {
         }
     }
 
+    if (getAlgorithm() == Algorithm::EltwisePowerStatic && getOriginalInputPrecisionAtPort(0) == ov::element::bf16) {
```

Comment:
LGTM. minor suggestions: although it's scalar process, should not have significant extra time cost here. but how about we handle this gamma saturation during Eltwise Constructor function?